tesseract_geometry/cmake/Findassimp.cmake

```diff
@@ -1,3 +1,7 @@
+find_package(assimp QUIET CONFIG)
+
+if(NOT assimp_FOUND)
```

Comment:
Should this also check if the target `assimp::assimp` exists and if not create even if found?

Reply:
Good idea, I will add the check